setup-scripts/log-checker.py

```diff
@@ -76,23 +76,11 @@ async def check_docker_logs():
         if len(std_err) > one_mb:
             pos = std_err.find("\n", -one_mb)
             std_err = std_err[pos+1:]
-        upload_name = "{}-{}-{}-{}-{}:{}:{}_err.log".format(CONTAINER_NAME, time.year, time.month, time.day, time.hour, time.minute, time.second)
-        await send_msg(client, "Error(s) found in log!", file=discord.File(io.BytesIO(std_err.encode()), filename=upload_name), force_notify=True)
-        # Send at most DISCORD_MAX_MESSAGE_LENGTH characters of logs, rounded
-        # down to the nearest new line. This is a limitation in the size of
-        # Discord messages - they can be at most 2000 characters long (and we
-        # send some extra characters before the error log).
-        if len(std_err) > DISCORD_MAX_MESSAGE_LENGTH:
-            pos = std_err.find("\n", -DISCORD_MAX_MESSAGE_LENGTH)
-            std_err = std_err[pos+1:]
-        await send_msg(client, "Error(s) preview:\n{}".format(std_err), force_notify=True)
-        return
+            return await send_msg(client, "Error(s) found in log!", file=std_err, force_notify=True)
 
     # If there are any critical or severe errors. upload the whole log file.
     if 'Critical' in std_out or 'Severe' in std_out or 'panic' in std_out:
-        upload_name = "{}-{}-{}-{}-{}:{}:{}.log".format(CONTAINER_NAME, time.year, time.month, time.day, time.hour, time.minute, time.second)
-        await send_msg(client, "Critical or Severe error found in log!", file=discord.File(io.BytesIO(std_out.encode()), filename=upload_name), force_notify=True)
-        return
+        return await send_msg(client, "Critical or Severe error found in log!", file=std_out, force_notify=True)
```

Comment:
Um... Same here. I see you're importing `upload_to_skynet` but I don't see where it's used. I might be missing it (first coffee) but I think it's not there.

Reply:
passing `file=std_out` to `send_msg` will upload the log and append the skylink in the message